aodntools/timeseries_products/Documentation/aggregated_timeseries.md

```diff
@@ -0,0 +1,175 @@
+All output variables with the OBS dimension are sorted in chronological order.
+
+In order to keep track of the provenance of VoI in the aggregated file, accessory variables are created:
+
+`instrument_index(OBSERVATION)`: index [0:number of files] of the instrument used, referencing the `INSTRUMENT` dimension.
+
+`source_file(INSTRUMENT, string256)`: URLs of the files used.
+
+`instrument_id(INSTRUMENT, string256)`: concatenated deployment_code, instrument and instrument_serial_number from the global attributes of each file.
+
+`LATITUDE(INSTRUMENT)`: LATITUDE per instrument.
+
+`LONGITUDE(INSTRUMENT)`: LONGITUDE per instrument.
+
+`NOMINAL_DEPTH(INSTRUMENT)`: nominal depth per instrument.  
+
+
+### Attributes
+
+The variable attributes will comply with the IMOS metadata standards.
+
+The global metadata will be a set of IMOS standard attributes. Fixed attributes are read from a JSON file that contains the {key:value} pairs for each of them. See the contents of this file at the end of this document.
+
+Attributes specific to each aggregated product, are added as follows:
+
+`site_code`, `local_time_zone`: obtained from the input files (should be the same in all of them);
+
+`time_coverage_start`, `time_coverage_end`: set to the full range of `TIME` values in the aggregated file;
+
+`geospatial_vertical_min/max`: set to the full range of `DEPTH` values in the aggregated file;
+
+`geospatial_lat_min/max`: set to the full range of `LATITUDE` values in the aggregated file;
+
+`geospatial_lon_min/max`: set to the full range of `LONGITUDE` values in the aggregated file;
+
+`date_created`: set to the date/time the product file is created;
+
+`history`: set to “<date_created>: Aggregated file created.”;
+
+`keywords`: set to a comma-separated list of the main variable names (“<VoI>, TIME, DEPTH, LATITUDE, LONGITUDE”);
+
+`lineage`: a statement about how the file was created, including a link to the code used, and any input parameters (except the input files, which are listed in the source_file variable);
+
+`title`: "Aggregated time-series product: <VoI> at <sit````e_code> between <time_coverage_start> and <time_coverage_end>"
+
+## Output
+
+
+The output from a single run of the code will be an aggregated file of all available measurements of a single non-velocity variable at one mooring site.
+
+The product will be delivered, in netCDF4 format, compliant with the CF-1.6 and IMOS-1.4 conventions, and
+structured according to the [indexed ragged array representation](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#_indexed_ragged_array_representation).
+
+## Sample Files
+
+Sample files could be downloaded from AODN THREDDS(add link) server.
+
+
+## Distribution
```

Comment:
Distribution, discovery and access, and maintenance are not relevant to the code, so I would remove them too.

Reply:
eliminated